tsrc/test/cli/test_push.py

```diff
@@ -31,9 +25,41 @@ def test_push_use_given_push_spec(
 ) -> None:
     tsrc.git.run(repo_path, "checkout", "-b", "local")
     push_args.push_spec = "local:remote"
-    repository_info = tsrc.cli.push.RepositoryInfo(repo_path)
-    dummy_push = DummyPush(repository_info, push_args)
+    repository_info = tsrc.cli.push.RepositoryInfo(mock_workspace_git_urls(), repo_path)
+    dummy_push = PushAction(repository_info, push_args)
     dummy_push.push()
     _, out = tsrc.git.run_captured(repo_path, "ls-remote")
     assert "local" not in out
     assert "heads/remote" in out
+
+
+def test_service_from_url() -> None:
+    workspace_mock = mock_workspace_git_urls()
+
+    assert "github" == tsrc.cli.push.service_from_url("git@github.com:TankerHQ/tsrc.git", workspace_mock)
```

Comment:
Small nitpick: We usually write assertions in the form 'assert actual == expected'.

```
assert  tsrc.cli.push.service_from_url("git@github.com:TankerHQ/tsrc.git", workspace_mock) == "github"
```